former-developers.md

```diff
@@ -0,0 +1,19 @@
+The SuperDARN Radar Software Toolkit (RST) was originally written by R.J Barnes and K. Baker at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Since then, members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. Below is a list of contributors to the RST. If you believe someone is missing from this list, please contact the Data Analysis Working Group <darn-dawg@isee.nagoya-u.ac.jp>.
+
+- D. Andre, `fitacf`
+- K. Baker, original JHUAPL author
```

Comment:
While I can't speak to the full development history, if anyone should receive primary credit for the original FITACF algorithm it should probably be Kile.

Reply:
```suggestion
- K. Baker, original JHUAPL author, `fitacf1.0`
```